src/tsf/TfCtxtComp.h

```diff
@@ -26,13 +26,7 @@ Revision History:
 /////////////////////////////////////////////////////////////////////////////
 // CCompString
 
-class CCompString : public CComBSTR
-{
-public:
-    CCompString() : CComBSTR() { };
-    CCompString(__in_ecount(dwLen) LPWSTR lpsz, DWORD dwLen) : CComBSTR(dwLen, lpsz) { };
-    virtual ~CCompString() { };
-};
+using CCompString = std::wstring;

```

Comment:
If we're doing this, could we just replace all the `CCompString`'s with `std::wstring`?

Reply:
That would be good. While it's tempting to keep the `CCompString` name around in order to minimize changes it also makes it more difficult to see what parts of the codebase are actually affected by this change. 

For example, if you go and change everything to `std::wstring` you may notice that in `CEditSessionCompositionComplete::CompComplete` there's a `wstring` object that is created and then passed as a view - but the view can be created directly.

Reply:
Yes, please do not use this alias. The `CCompString` name implies it is coming from ATL or COM, so it's very misleading to redefine it as a `std::wstring`. Please just find/replace.

Reply:
I changed `CEditSessionCompositionComplete::CompComplete` to
```
hr = conv_area->DrawResult({ wstr.get(), cch });
```